plugins/CoreAdminHome/stylesheets/generalSettings.less

```diff
@@ -74,12 +74,31 @@ table.admin  tbody td:hover, table.admin  tbody th:hover {
     padding: 15px;
 }
 
+.admin p + h2 {
+  margin-top: 35px;
+  // .admin p defines a margin-bottom of 10px, we make sure we still have a margin-top of 45px this way
+}
+
 .admin h2 {
-    border-bottom: 1px solid #DADADA;
-    margin: 15px -15px 20px 0;
+    border-bottom: 0px;
+    margin: 45px -15px 11px 0;
     padding: 0 0 5px 0;
     font-size: 24px;
     width:100%;
+
+  &:first-of-type:not(.secondary) {
+    margin-top: 17px;
+    margin-bottom: 16px;
+    border-bottom: 1px solid #DADADA;
+  }
+}
+
+.admin h2 + h3 {
+  margin-top: 0px;
+}
+
+.admin h3 {
+  margin-top: 30px;
 }
```

Comment:
It would be better to make headers work correctly everywhere in Piwik rather than just in the admin section. Else we end up with different styles in different parts of Piwik and we need to spend time and effort fixing all this (e.g. what I started those last weeks)